code/game/objects/structures/memorial.dm

```diff
@@ -20,3 +19,27 @@ This memorial has been designed for him and any future coders to perish.
 	density = TRUE
 	anchored = TRUE
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+
+
+/*
+
+In Memoriam 26th August, 2020
+Our friend, fellow admin, and champion of the "short" reply, OrdoDictionary passed away recently.
+While this news was not unexpected - Ordo had been fighting bravely against prion disease for some time - it remains as heartbreaking as it is unfair.
+Ordo was sharp as a razor, with wit to match. During the time we knew him he was pursuing a PhD, yet he still found time to volunteer here and help his fellow spessman.
+In admin channels you were as likely to find Ordo pontificating on history or food or ham radio as you were anything related to the game. But you knew -
+no matter the topic - that his opinion was informed and valuable, and that he would deliver it with nuance, class and charm.
+
+This community - and the world - is worse off without him in it.
+He will be missed.
+
+So should you read this and should you get the chance, raise a glass in his memory.
+…but be sure to break out the good stuff.
+That’s how he would have wanted it.
+*/
+/obj/item/clothing/accessory/medal/gold/ordom
```

Comment:
I know this is technically misfiled, but I figured we'd keep these important objects separate from the rest of the code so there's no accidental deletions.